instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/ProcessArguments.java

```diff
@@ -5,8 +5,11 @@
 
 package io.opentelemetry.instrumentation.resources;
 
+import javax.annotation.Nonnull;
+
 final class ProcessArguments {
 
+  @Nonnull
```

Comment:
Nit: we only use `@Nullable`, everything not annotated with anything should be non-null by default.

Reply:
Ok, removed.